src/picologging/logrecord.hxx

```diff
@@ -45,7 +45,7 @@ _PyTime_t current_time();
 
 
 extern PyTypeObject LogRecordType;
-#define LogRecord_CheckExact(op) Py_IS_TYPE(op, &LogRecordType)
+#define LogRecord_Check(op) PyObject_TypeCheck(op, &LogRecordType)
```

Comment:
The first one should not be removed, just have both. It's convention to have 2 macros.

XXX_CheckExact does a Py_IS_TYPE (fastest, pointer comparison)
XXX_Check does a PyObject_TypeCheck (checks pointer first then does a MRO scan which is slower)

Reply:
Done.